modules/assim.sequential/R/sda.enkf.R

```diff
@@ -261,14 +260,14 @@ sda.enkf <- function(settings, obs.mean, obs.cov, pick.trait.params = NULL,
     if(any(obs)){ #if no observations skip analysis
     mu.f = as.numeric(apply(X,2,mean,na.rm=TRUE))
     Pf   = cov(X)
-    Y    = obs.mean[[t]][pmatch(colnames(X), names(obs.mean[[t]]))]
+    Y    = na.omit(obs.mean[[t]][pmatch(colnames(X), names(obs.mean[[t]]))])
```

Comment:
This won't work as coded. If you do a na.omit on Y, then Y and R won't line up anymore. You need to detect the NA's in Y and remove those rows & columns from R as well.  You should also throw a warning because Y shouldn't have NAs in it.